SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -1,20 +1,205 @@
+                HStack(spacing: Constants.spacingButton) {
+                    buyButton
+                    shareButton
+                }
+            }
         }
+    }
+    
+    // MARK: Private Properties
+    
+    private let cars = [
+        Car(name: "Пятера",
+            imageName: "11",
+            toning: "Аквариум",
+            enginePower: 74,
+            price: 25000),
+        Car(name: "Не пешком",
+            imageName: "22",
+            toning: "Гаишники попутают",
+            enginePower: 82,
+            price: 30000),
+        Car(name: "Дедушкина",
+            imageName: "33",
+            toning: "Аквариум",
+            enginePower: 123,
+            price: 15000),
+        Car(name: "Феррари",
+            imageName: "44",
+            toning: "Аквариум",
+            enginePower: 162,
+            price: 100000)
+    ]
+    
+    @State private var selectionCarPicker = Constants.zeroNumber
+    @State private var selectionSalonPicker = Constants.zeroNumber
+    @State private var isShare = false
+    @State private var isBuy = false
+    @State private var isSteeringWheel = false
+    @State private var underestimate = Constants.underestimateNumber
+
+    private var salonTypes = Constants.salonTypes
+    
+    private var showroomNameText: some View {
+        Text(Constants.showroomNameText)
+            .bold()
+            .font(.system(size: Constants.showroomNameTextFontSize))
+            .foregroundColor(.red)
+    }
+        
+    private var buyButton: some View {
+        return Button {
+            isBuy = true
+        } label: {
+            Text(Constants.buyText).font(.system(size: Constants.buiButtonFontSize))
+                .padding()
+        }.alert(isPresented: $isBuy) {
+            Alert(title: Text(Constants.buyButtonAlertTitleText), message: Text(Constants.buiButtonAlertMessageText), primaryButton: .cancel(), secondaryButton: .default(Text(Constants.buiButtonAlertDefaultText)))
+        }.frame(width: Constants.buiButtonWidthNumber, height: Constants.buiButtonHeightNumber)
+            .foregroundColor(.black)
+            .background(Color.red)
+            .cornerRadius(Constants.buiButtonCornerRadiusNumber)
+    }
+    
+    private var carImage: some View {
+        Image(cars[selectionCarPicker].imageName)
+            .resizable()
+            .frame(width: Constants.carImageWidthNumber,
+                   height: Constants.carImageHeightNumber)
+            .cornerRadius(Constants.carImageCornerRadiusNumber)
+            
+    }
+    
+    private var salonPicker: some View {
+        Picker(Constants.salonPickerText, selection: $selectionSalonPicker) {
+            ForEach(Constants.zeroNumber ..< salonTypes.count) {
+                                 Text(salonTypes[$0])
+                             }
+                         }
+                         .pickerStyle(.navigationLink)
+                     }
+            
+    private var carPicker: some View {
```

Comment:
```suggestion
    private var carPicker: some View {
```
везде правь View в конце